BookingProject/BookingWebAPI/BookingWebApi.Domain/Entities/ProcessedEvent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Domain.Entities
+{
+    public class ProcessedEvent
+    {
+        public string ProcessedEventId { get; set; }
+        public DateTime ProccesedAt { get; set; } = DateTime.UtcNow;
+    }
```

Comment:
Can we move it to infrastructure? I don't think it's related to domain.
+ we can remove ProcessedAt bc idk why we need it. We can get the date of consumption of the event from kafka record. If it might be useful please explain, thanks